modules/swagger-jaxrs2/src/main/java/io/swagger/v3/jaxrs2/Reader.java

```diff
@@ -374,19 +374,14 @@ public OpenAPI read(Class<?> cls,
         globalParameters.addAll(ReaderUtils.collectFieldParameters(cls, components, classConsumes, null));
 
         // iterate class methods
```

Comment:
Reflection.getMethods just returns the desired methods. No further filtering should be done on overriding.